airflow/plugins/operators/airtable_to_gcs.py

```diff
@@ -71,7 +71,7 @@ def fetch_from_airtable(self, api_key):
             f"Downloading airtable data for {self.air_base_name}.{self.air_table_name}"
         )
         all_rows = Table(api_key, self.air_base_id, self.air_table_name).all()
-        self.extract_time = pendulum.now("utc")
+        self.extract_time = pendulum.now()
```

Comment:
wait so can we clarify the impact of this -- this is going to have `Z` timestamps? why are we doing this / why are we doing it here?

Reply:
This _removes_ the explicit UTC timestamp i.e. no more Z but +00:00. This is the only place with an explicit Z and the problem is that pendulum doesn't handle the UTC properly anyways.

Reply:
is this going to mess us up later for type 2 tables then (like, do Airtable external tables have wrong times)? just want to understand impact

Reply:
That's a good point, we should test how bigquery parses the timestamps. At worst we can fix the keys, but I think it should be fine. They're all UTC it just depends on whether the serialization used Z vs a 0 offset (I'm fuzzy on the difference, to be honest.)

Reply:
I guess I meant more "pendulum doesn't handle the UTC properly anyways" -- do we think things are substantively wrong...?

Reply:
No I think this is just a serialization issue. `datetime` picks up that they are UTC just fine.